programming.md

```diff
@@ -165,7 +161,6 @@ Website - https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa4ad5JISViSb2FVG8Vwa
 
 [![JUnit5 Basics by Java Brains](https://user-images.githubusercontent.com/1084664/66758571-24318c00-eebc-11e9-8457-eea82dc64e3f.png)](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa4ad5JISViSb2FVG8Vwa4o)
 
-
```

Comment:
Thanks for the updates. Can you also remove these changes? These are also done by prettier and this is not consistent with other files (we use 2 empty lines to separate sub-topics).

Basically any other change except adding the resource is not needed.